core/src/core_icu.cpp

```diff
@@ -49,16 +48,20 @@ km_core_usv *unicode_string_to_usv(icu::UnicodeString& src) {
  * @return false if failure
  */
 bool normalize(const icu::Normalizer2 *n, std::u16string &str, UErrorCode &status) {
```

Comment:
Why are we moving things to `std::u32string` when `std::u16string` is what we're using on the primary `normalize` method?

Reply:
Actually, most processing is in UTF 32… The 16 bit stuff was only because that's ICUs native API…